messaging/src/main/java/com/iluwatar/messaging/MessagingApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.iluwatar.messaging;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
```

Comment:
Here, above the main class, would be a perfect place to describe the pattern briefly and explain how this example implements it.

Reply:
Added docs here